src/lock.js

```diff
@@ -23,6 +23,16 @@ module.exports = class Lock {
           body: lockComment
         });
       }
+      
+      if (lockLabels) {
```

Comment:
Not sure if it should be plural or not. It accepts an array of strings so I did that even though personally I would just simply call it `outdated`. I did send a request for an `outdated` lock reason already so this might not be necessary when that happens later.

Reply:
I think being able to set a single label might be enough, since the primary reason for it would be to easily list issues locked by the bot.